script.js

```diff
@@ -1,6 +1,129 @@
+const workCards = document.querySelector('.work-container');
+// const cards = document.querySelectorAll(".work-container > div")
+const body = document.querySelector('body');
 const hamburger = document.getElementById('handberga-menu');
 const menu = document.getElementById('menu');
 const existBtn = document.getElementById('exit');
+const projectCards = [
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    image: 'Images/abvout.png',
+    Tech: ['Html', 'CSS', 'Bootstrap', 'Ruby'],
+    button: 'See project',
+  },
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    image: 'Images/abvout.png',
+    Tech: ['Html', 'CSS', 'Bootstrap', 'Ruby'],
+    button: 'See project',
+  },
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    image: 'Images/abvout.png',
+    Tech: ['Html', 'CSS', 'Bootstrap', 'Ruby'],
+    button: 'See project',
+  },
+  {
+    Name: 'Profesional Art Printing Data ',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    image: 'Images/abvout.png',
+    Tech: ['Html', 'CSS', 'Bootstrap', 'Ruby'],
+    button: 'See project',
+  },
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    image: 'Images/abvout.png',
+    Tech: ['Html', 'CSS', 'Bootstrap', 'Ruby'],
+    button: 'See project',
+  },
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    image: 'Images/abvout.png',
+    Tech: ['Html', 'CSS', 'Bootstrap', 'Ruby'],
+    button: 'See project',
+  },
+];
+
+projectCards.forEach((card) => {
+  const content = `
+  <div>
+  <h3>${card.Name}</h3>
+
+  <p>${card.Description}</p>
+
+  <ul>
+      <li>${card.Tech[0]}</li>
+      <li>${card.Tech[1]}</li>
+      <li>${card.Tech[2]}</li>
+  </ul>
+  <button id="sampleproject" type="button">${card.button}</button>
+</div>
+  `;
+  workCards.innerHTML += content;
+});
```

Comment:
- [ ] You did a great job on the design but for this project, you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design as much as possible. Please fix the below issues for the desktop screen and make sure your design matches the Figma design.
  - Please reduce the font size of the project title and description.
  - Please reduce the gap between the project title and technologies.
  - Please increase the width of the model so that the project description is aligned with the image.
  - Please apply the blur effect in the background. `hint: backdrop-filter: blur(Npx);` N is referred to as a number for pixels.

### Your design
![image](https://user-images.githubusercontent.com/83803414/189077243-cb13bfd6-dd1d-489f-85c4-24466fa25dc6.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/189077547-3f2f0629-de07-4be0-a5e7-ff7eea1aff90.png)